content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,232 @@
+
+- #1 Exists outside of the O3DE Editor
+  - Use in generic tests which launch applications and send external signals
+  - Helps and monitor for application crashes
+  - Test file names should start with `test_`
+  - Same Python interpreter that launches when you run `o3de/python/python.sh` or `o3de\python\python.cmd`
+
+- #2 Embedded inside the O3DE Editor
+  - Use in "EditorPythonBindings" tests which target workflows inside the editor
+  - Helps expose editor functionality to a test script
+  - Test file names should ***not*** include affixes such as `test_` or `_tests`
+  - PyTest is still initiated by the external python, to handle editor crashes
+
+### Registering a new Python Test
+
+Registering a Python-based test is simpler than registering a C++ test. However it similarly requires you define the library that will be tested. The steps below assume you have already defined a production library. You can read more about defining production code in [Getting Started with the CMake Build System](/docs/user-guide/build/).
+
+#### Step 1: Register a PyTest Target
+
+Find the CMakeLists.txt that defines the system you are testing. It should exist at a path similar to `o3de/.../<MyModule>/CMakeLists.txt` . Note that it may instead be appropriate to register broader tests in a parent directory's CMakeLists.txt. If you are creating a test that is specific to a game project, you should register it in a CMakeLists.txt in that game project. When you find the right place for the test to be registered, add a line similar to:
+
+```
+ly_add_pytest(
+    NAME myPythonTest 
+    PATH ${CMAKE_CURRENT_LIST_DIR}/Tests/test_MyFile.py
+    TEST_SERIAL # many larger-than-unit tests can interfere with one another in parallel
+    RUNTIME_DEPENDENCIES # helps the test build-and-run from an IDE
+        Legacy::Editor
+        AZ::AssetProcessor
+        AutomatedTesting.Assets
+)
+```
+
+To verify everything is set up correctly, run the [CMake configure command](/docs/user-guide/build/configure-and-build/) from the CMake CLI or CMake GUI. This will register everything you just added, and emit errors if anything was misconfigured.
+
+#### Step 2: Write new Python tests
+
+These tests primarily use standard [Python](https://docs.python.org/3/) code along with [PyTest](https://docs.pytest.org/). Depending on where your test is focused, it likely wants to include one of two automation libraries:
+
+Internal tests of the O3DE Editor functionality should use [EditorPythonBindings](/docs/user-guide/testing/parallel-pattern/). This helps expose C++ interfaces of the Editor to Python. These tests almost always require the `TEST_SERIAL` flag.
+
+External tests at the Operating System level often use the [LyTestTools](/docs/user-guide/testing/lytesttools/) module, which simplifies interacting with O3DE applications. Tests using these tools are discussed more at . These tests often use the `TEST_SERIAL` flag.
```

Comment:
```suggestion
External tests at the operating system level often use the [LyTestTools](/docs/user-guide/testing/lytesttools/) module, which simplifies interacting with O3DE applications. Tests using these tools are discussed more at . These tests often use the `TEST_SERIAL` flag.
```
> Tests using these tools are discussed more at . 

You wanted to include a link here.

Reply:
Desired documentation does not yet exist, deleting sentence.